test/cmd/legacy-script.sh

```diff
@@ -385,468 +386,475 @@ runTests() {
     kubectl get "${kube_flags[@]}" -f hack/testdata/kubernetes-service.yaml
   fi
 
-  #########################
-  # Kubectl version #
-  #########################
+   if [[ -n "${WHAT-}" ]]; then
+    for pkg in ${WHAT}
+    do 
+      record_command run_${pkg}_tests
+    done
+  else
+    #########################
+    # Kubectl version #
+    #########################
 
-  record_command run_kubectl_version_tests
+    record_command run_kubectl_version_tests
 
```

Comment:
Are all these indentation changes necessary?

Reply:
This was part of an if/else conditional on the precence of the `WHAT` argument.  I'll rework to escape early and keep the bottom of this with the same formatting